agentPresence/agentPresenceController.js

```diff
@@ -0,0 +1,87 @@
+var micc = new Micc('localhost');
+
+function login() {
+    micc = new Micc($('#miccServer').val());
+    micc.login(
+        $('#username').val(),
+        $('#password').val(),
+        connectToEmployeeHub
+    );
+}
+
+function connectToEmployeeHub(data) {
+    micc.getEmployeeStats('me', populateEmployeeInfo);
+    micc.getEmployeeBusyReasonCodes('me', populateBusyReasonCodes);
+
+    var miccSignalr = new MiccSignalR(data.miccServer, data.access_token);
+    miccSignalr.start(function () {
+        var hub = miccSignalr.createEmployeeHubProxy();
```

Comment:
I tried to make it more generic. This will allow us to have less functions in miccSignalr if we end up using too many hubs for samples. 
I can make it more specific just like you mentioned and in that case miccSignalr will keep reference for all hubs and use it in respective functions.